src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -20,6 +20,19 @@ internal static partial class SocketPal
         // PlatformNotSupportedException instead.
         public const int ProtocolInformationSize = 0;
 
+        public readonly static bool SupportsMultipleConnectAttempts = GetPlatformSupportsMultipleConnectAttempts();
+        private readonly static bool SupportsDualModeIPv4PacketInfo = GetPlatformSupportsDualModeIPv4PacketInfo();
```

Comment:
Ah, I missed the visiblity.  Given they serve a similar purpose, might actually make more sense to keep these consistent and named this why.  My care factor is low.